pkg/controller/plan/adapter/ocp/client.go

```diff
@@ -73,14 +73,14 @@ func (r *Client) PowerOff(vmRef ref.Ref) error {
 // PowerOn implements base.Client
 func (r *Client) PowerOn(vmRef ref.Ref) error {
 	vm := cnv.VirtualMachine{}
-	err := r.Client.Get(context.TODO(), client.ObjectKey{Namespace: vmRef.Namespace, Name: vmRef.Name}, &vm)
+	err := r.Destination.Client.Get(context.TODO(), client.ObjectKey{Namespace: vmRef.Namespace, Name: vmRef.Name}, &vm)
```

Comment:
i don't remember. but i do think it can be done if we post the vm yaml with power on status. it may need change in the builder if we don't fully copy the yaml.

Reply:
> i don't remember. 

in that case, I talked about it with @bennyz  :)

Reply:
yeah I think you are referring to this issue https://github.com/kubev2v/forklift/issues/437

Reply:
> yeah I think you are referring to this issue #437

right right